src/modalvolumeData_setup.py

```diff
@@ -0,0 +1,107 @@
+import modal
+import gdown
+import os
+import shutil 
+import logging
+import yaml
+import zipfile
+from dotenv import load_dotenv
+import boto3
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+image = modal.Image.debian_slim().run_commands(
+    "pip install gdown",
```

Comment:
Let's not do run_commands here and use `pip_install`, preferably add these to a requirements.txt file or requirements-modal.txt and reference that file. Modal has a way to install from requirements.txt (see docs).